legal-api/migrations/versions/e85053684b3c_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: e85053684b3c
+Revises: 92c4974ce96f
+Create Date: 2020-02-11 13:25:05.404822
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e85053684b3c'
+down_revision = '92c4974ce96f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, 'colin_event_ids', ['colin_event_id'])
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(None, 'colin_event_ids', type_='unique')
+    # ### end Alembic commands ###
```

Comment:
Note - this migration is unrelated to my work, but was generated by alembic based on the codebase. Functionally it changes nothing since the colin_event_id is already the primary key.